src/components/NewClusterModal.js

```diff
@@ -219,6 +219,22 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
     })
   }
 
+  sendUpdateMetrics(newRuntime, newPersistentDisk) {
+    const machineType = newRuntime.cloudService === cloudServices.GCE ? newRuntime.machineType : newRuntime.masterMachineType
+    const { cpu, memory } = findMachineType(machineType)
+
+    Ajax().Metrics.captureEvent(Events.cloudEnvironmentUpdate, {
+      ...extractWorkspaceDetails(this.makeWorkspaceObj()),
+      ...newRuntime,
+      cpu,
+      memory,
+      persistentDiskSize: newPersistentDisk?.size,
+      runtimeCostPerHour: Utils.formatUSD(runtimeConfigCost(this.getPendingRuntimeConfig())),
```

Comment:
Do we want to format these? It might be easier to compute with on the analytics side if we leave it as a number.